website/src/styles/global.css

```
@@ -0,0 +1,137 @@
+/* ── Design tokens — Catppuccin Mocha ──────────────────────────────────────── */
+:root {
+	--font-body: GenSenRoundedTW, "GenSenRounded2 TW", sans-serif;
```

Stylelint (17.9.0)

[error] Expected "GenSenRoundedTW" to be "gensenroundedtw" (value-keyword-case)

(value-keyword-case)